index.html

```diff
@@ -0,0 +1,255 @@
+
+<section class="about w-[70%] h-80 ml-40 bg-gradient-to-t from-slate-200 ">
+  <h1 class="about about--title text-center  font-bold text-6xl mt-[20%} ">
+    Are you ready to grow your business with us?
+  </h1>
+  <p class="about__text ml-56 mt-10 ">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.
+  </p><br><br>
+  <a href="#" class="about__link ml-96 mt-28 bg-black font-bold rounded-md text-white py-2 px-3">View Pricing</a>
+</section>
+<footer class="footer">
+  <div class="footer__container p-10 bg-gray-800 text-gray-200">
+    <div class="footer__row mx-w-7xl mx-auto">
+      <div class="footer__col grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-2">
+        <div class="footer__title mb-5">
+          <h4 class="footer__heading">Company</h4>
+          <ul class="footer__lists">
+            <li class="footer__link"><a href="" class="footer__item">About Us</a></li>
+            <li class="footer__link"><a href="" class="footer__item">Why Choose us</a></li>
+            <li class="footer__link"><a href="" class="footer__item">Pricing</a></li>
+            <li class="footer__link"><a href="" class="footer__item">Testimonial</a></li>
+          </ul>
+        </div>
+        <div class="footer__title mb-5">
+        <h4 class="footer__heading">Resources</h4>
+        <ul class="footer__lists">
+          <li class="footer__link"><a href="" class="footer__item">Privacy Policy</a></li>
+          <li class="footer__link"><a href="" class="footer__item">Terms and Condition</a></li>
+          <li class="footer__link"><a href="" class="footer__item">Blog</a></li>
+          <li class="footer__link"><a href="" class="footer__item">Contact Us</a></li>
+        </ul>
+      </div>
+      
+      <div class="footer__title mb-5">
+        <h4 class="footer__heading">Product</h4>
+        <ul class="footer__lists">
+          <li class="footer__link"><a href="" class="footer__item">Project managment</a></li>
+          <li class="footer__link"><a href="" class="footer__item">Time tracker</a></li>
+          <li class="footer__link"><a href="" class="footer__item">Time schedule</a></li>
+          <li class="footer__link"><a href="" class="footer__item">Lead generate</a></li>
+          <li class="footer__link"><a href="" class="footer__item">Remote Collaboration</a></li>
+
+        </ul>
+      </div>
+      <div class="subscibe">
+        <img src="images/logo-white.png" class="subscibe__logo" />
+        <p class="subscibe__text">Subscribe to our Newsletter</p>
+        <div class="subscibe__form">
+            <input class="subscribe__input" type="text" placeholder="Enter Your Email">
+            <button class="button button--background">Subscribe</button>
+        </div>
+    </div>
+    
+  </div>
+  </div>
+  <div class="footer-end text-center w-full ">
+    <hr class="footer-end__line "/>
+    <p class="footer-end__text text-center">Copyright Finsweet 2022</p>
+    <hr class="footer-end__line "/>
+  </div>
```

Comment:
Inside the footer block, having a block called footer_end is confusing. It sounds like it should be an element of footer. If it is a block, try to change the name.